Dockerfile

```diff
@@ -1,31 +1,39 @@
-FROM python:3.8-slim-buster as builder
+FROM python:3.9-slim as base
 
 ENV LANG=C.UTF-8 \
     DEBIAN_FRONTEND=noninteractive \
     PIP_NO_CACHE_DIR=true
 
-RUN apt-get update && apt-get install --no-install-recommends -qq -y gcc g++ && apt-get clean autoclean \
-  && apt-get autoremove -y \
-  && rm -rf /var/lib/apt/lists/* \
-  && rm -f /var/cache/apt/archives/*.deb
+RUN apt-get update && apt-get install -y --no-install-recommends -qq gcc=4:10.2.1-1 libffi-dev=3.3-6 g++=4:10.2.1-1 git=1:2.30.2-1 curl=7.74.0-1.3+deb11u1 \
```

Comment:
Are extra packages needed here for sure?

Reply:
Removed